src/Main.java

```diff
@@ -0,0 +1,235 @@
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        int h = 2;
+        int b = 0;
+     while (h == 2) {
+         b =Gamble(b);
+
+     }
+    }
+
+    public static int Gamble(int money) {
+        Scanner pickGame = new Scanner(System.in);
+        System.out.println("You had " + money + " dollars from the last round. Would you like to keep that balance, or get some other amount of money? Enter 1 to keep that balance, enter 2 to change it.");
+        if (pickGame.nextInt() == 1) {
+            System.out.println("Okay. You will start with $" + money);
+        } else {
+            System.out.println("How much money do you want to start with?");
+            money = pickGame.nextInt();
+        }
+
+        System.out.println("Pick a game to play.");
+        System.out.println("Roulette Wheels - 1");
+        System.out.println("Slot Machines - 2");
+        System.out.println("Lottery - 3");
+        int gamePicked = pickGame.nextInt();
+        if (gamePicked == 1) {
+            return(playRoulette(money));
```

Comment:
I like you have made functions for each of the game options (roulette / slot / lottery).